samples/AspNetCoreSseServer/SseServerStreamTransport.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Buffers;
+using System.Net.ServerSentEvents;
+using System.Text.Json;
+using System.Threading.Channels;
+using McpDotNet.Protocol.Messages;
+using McpDotNet.Protocol.Transport;
+using McpDotNet.Utils.Json;
+
+namespace AspNetCoreSseServer;
+
+public class SseServerStreamTransport(Stream sseResponseStream) : ITransport
```

Comment:
Is the thinking this is a sample for right now but we'll subsequently look at making this real in a separate package?